packages/wonder-blocks-data/util/response-cache.js

```diff
@@ -6,6 +6,8 @@ function deepClone<T: {...}>(source: T | $ReadOnly<T>): $ReadOnly<T> {
      * We want to deep clone the source cache to dodge mutations by external
      * references. So we serialize the source cache to JSON and parse it
      * back into a new object.
+     *
+     * NOTE: This doesn't work for get/set property accessors.
```

Comment:
Does this mean that people shouldn't try to cache things with get/set properties?  It might be good to put this in the docs if it isn't already there.

Reply:
Yeah, though this is a very common thing for stuff that gets serialized in JavaScript; get/set properties won't be included in the serialization. And it will be very noticeable.

Reply:
Hard to imagine a use-case where folks would do that.